zuul.d/run.yaml

```diff
@@ -22,3 +22,37 @@
       command: "ansible-playbook -i inventories/uc-engine test.yml"
       args:
         chdir: "{{ zuul.project.src_dir }}"
+
+    - name: Reset pg
+      become: yes
+      shell: "sudo -u postgres psql"
+      args:
+        stdin: |
+          SELECT pg_terminate_backend(pg_stat_activity.pid)
+          FROM pg_stat_activity
+          WHERE pg_stat_activity.datname = 'asterisk'
+          AND pid <> pg_backend_pid();
+
+    - name: Reset engine
+      become: yes
+      shell: "wazo-reset -f"
+
+    - name: create acceptance config
+      blockinfile:
+        create: yes
+        path: ~/.wazo-acceptance/config.yml
+        block: |
+          default:
+            wazo_host: localhost
+            debug:
+              global: True
+              
+              acceptance: True
+              linphone: False  # otherwise, there are unprintable (^H) character that jenkins cannot decode in junit.xml
+            setupd:
+              timeout: 120
```

Comment:
this timeout is now useless, the default has been increased to 60 seconds (I removed it from jenkins config)

https://github.com/wazo-pbx/wazo-setupd-client/blob/master/wazo_setupd_client/commands/setup.py#L7